test/distributed/test_functional_api.py

```diff
@@ -57,6 +69,7 @@ def new_subgroups(group_size: int, pg_tag=None):
     return cur_subgroup, subgroups
 
 
+@skipIfHpu
```

Comment:
Is there a way to avoid having one skip line per device type?
Something like: `@skipIfDeviceIs(["hpu", ...])` ?

Reply:
This is a great suggestion. I suspect there are many such decorators and utilities which are specific to one device. 

I intend to raise a second PR to follow up on this in which I will address not only the changes to this decorator but other similar functionalities as well.

Reply:
Sounds good.